src/edgepi/adc/adc_multiplexers.py

```diff
@@ -27,8 +26,7 @@ def _format_mux_values(mux_p: CH, mux_n: CH):
 
     return mux_p_val, mux_n_val, mask
 
-
-def generate_mux_opcodes(mux_updates: dict):
+def generate_mux_opcodes(adc1_reg, adc1_mux, adc2_reg, adc2_mux):
```

Comment:
please edit the function description.

Reply:
Nice catch! https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/421/commits/b4fcbe436dca48f6ac756ec78a546931ec9815dc 

I'll check the rest of the function descriptions too.